includes/Login.php

```diff
@@ -6,10 +6,15 @@
 
 if( isset( $_SESSION['Username'] ) )
 {
+	
 	if( $_SESSION['Username'] == 'admin' )
 	{
 		header("Location:includes/dashboard/Admin.php");
 	}
+	else if( isset( $_SESSION['PostName'] ) )
+	{
+		header("Location:includes/dashboard/PostDashBoard.php");
+	}
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after closing brace; newline found

<!-- Comment by [SideCI](https://sideci.com) -->